Yoast/yoast-components
# Add explanations for input fields

This adds explanations to the input fields if provided. Right now only titles and meta's have the an explanation.

To test this:
- Go to step 9 'title settings'
- Check if explanations are shown below the input field for the title and the choice for the title separator.

Fixes https://github.com/Yoast/wordpress-seo/issues/5571
